jerry-core/jcontext/jcontext.h

```diff
@@ -154,6 +158,9 @@ typedef struct
 
 #endif /* !JERRY_GET_CURRENT_INSTANCE */
 
+#define JERRY_INIT_CURRENT_INSTANCE() \
+ struct jerry_instance_t *__current_external_instance = JERRY_GET_CURRENT_INSTANCE ();
```

Comment:
- Style: tabulation is two spaces, not one.
- Style: variable declaration in the declaration of this macro should _not_ be terminated with a semicolon to enforce that the invocation of the macro is written as `JERRY_INIT_CURRENT_INSTANCE  ();`
- Style: I cannot recall using double underscore prefixed identifiers anywhere in the project. Actually, that may even be against the C99 standard (which reserves such identifiers either for the standard or for the compiler implementation, if I'm not mistaken). A variable named like `jerry_current_external_instance`,  `jerry_current_instance`, or `jerry_external_instance` might work.